test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -5,6 +5,161 @@
  */
 #include "serializer.h"
 
+#include <stack>
+#include <string>
+
+namespace {
+
+/**
+ * Class that controls id creation for counters. Basically, a global variable
```

Comment:
`id_` is a part of class, not a global, right?

Reply:
I changed the comment